index.bs

```diff
@@ -543,18 +543,21 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
 	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+		<em>This section is non-normative.</em>
 
-			These data types MUST NOT be sanitized by UAs:
+			These data types MAY NOT be sanitized by UAs:
 
 			* image/png
 
 			* [=optional unsanitized data types=]
 
-			<dfn>Optional unsanitized data types</dfn> are [=representation/mime type=]s specified by the web authors that MUST NOT be sanitized by the user agent.
+			<dfn>Optional unsanitized data types</dfn> are [=representation/mime type=]s specified by the web authors that MAY NOT be sanitized by the user agent.
```

Comment:
The original text before https://github.com/w3c/clipboard-apis/pull/197 was
> The user agent, MAY sanitize representation ’s[ data ](https://pr-preview.s3.amazonaws.com/w3c/clipboard-apis/197/ad65071...0312e3e.html#representation-data), if it satisfies the below conditions:
>     1. representation ’s[ MIME type ](https://pr-preview.s3.amazonaws.com/w3c/clipboard-apis/197/ad65071...0312e3e.html#representation-mime-type)'s essence is "image/png", which should remain unsanitized to preserve meta data.

Dose it mean we have `image/png` to not  sanitize `image/png` data? If so, use "MAY NOT" for `image/png` might cause confusion.

Reply:
Changed it to mention that `image/png` MUST NOT be sanitized by UAs (per the original text).